cluster/latest/kubevirt-ssp-operator.yaml

```diff
@@ -177,7 +181,7 @@ spec:
         type: Recreate
       containers:
         - name: kubevirt-ssp-operator
-          image: quay.io/fromani/kubevirt-ssp-operator-container:latest
+          image: quay.io/shwetaap/kubevirt-ssp-operator-container:latest
```

Comment:
after you change the `IMAGE_REGISTRY` variable, please regenerate these manifests